cmd/end2endtest/helpers.go

```diff
@@ -631,3 +631,53 @@ func cspGenProof(pid, voterKey []byte, csp *ethereum.SignKeys) (*apiclient.Censu
 
 	return &apiclient.CensusProof{Proof: caProofBytes}, nil
 }
+
+func (t *e2eElection) verifyVoteCount(startTime time.Time, nvotesExpected int) error {
```

Comment:
i'd not pass startTime and instead set 	`startTime := time.Now()` inside verifyVoteCount. it does change the current behaviour but i believe it's reasonable to have each step have it's own timeout. step sendVotes, step verifyVoteCount, step endElectionAndFetchResults.

to be honest the current `c.timeout` is pretty useless (5 minutes, in start_test.sh never happens since there's a "wrapper" timeout that happens before) so we could refactor it:
```
	flag.DurationVar(&c.timeout, "timeout", apiclient.WaitTimeout, "timeout duration of each step")
```